nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/nifi.properties

```diff
@@ -93,7 +87,6 @@ nifi.content.repository.encryption.key=${nifi.content.repository.encryption.key}
 
 # Provenance Repository Properties
 nifi.provenance.repository.implementation=${nifi.provenance.repository.implementation}
-nifi.provenance.repository.debug.frequency=${nifi.provenance.repository.debug.frequency}
```

Comment:
Should `nifi.web.war.directory` also be removed from this file (as removed from `nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java`)?

Reply:
Good catch, I'm going to make that update and get this merged